src/common/components/Numpad/Numpad.tsx

```diff
@@ -58,8 +58,13 @@ export default forwardRef<HTMLDivElement, TProps>(function Numpad(
         className,
       )}
     >
+      <div className='input h-8 px-2 mb-1 w-[112px] flex justify-between items-center text-gray-a0 font-medium'>
+        {value.split('').map(item => (
+          <span>{item}</span>
+        ))}
+      </div>
       <input
-        className='input h-8 px-2 mb-1 w-[112px]'
+        className='input h-8 px-2 mb-1 w-[112px] hidden'
```

Comment:
It seems this input now become redundant since it is hidden, could you remove it?

Reply:
fixed